app/components/ChainsDropdown/ChainsDropdown.tsx

```diff
@@ -49,8 +35,7 @@ const ChainsDropdown = ({ chains, onChange, width }: ChainsDropdownProps) => {
       searchable
       aria-label="Add new"
       data={selectChainData}
-      icon={<LuSearch size={18} />}
-      itemComponent={SelectItem}
```

Comment:
why dont we need this anymore?

Reply:
Yeah, it's not customized.. it's a simple select